### PR TITLE
fix [unit_converter]: Update unit_converter to API v2.1

### PR DESCRIPTION
**Pull request by Pete-Hamlin**
Updated the `unit_converter` python extension to work with API 2.1. Also added support for a `GlobalQueryHandler` via the settings.

@DenverCoder1 I added myself as a maintainer as I'm happy to keep an eye on this one in case of breaking API changes. As this is your work though I'm happy to remove myself if you would prefer it.

### CODE REVIEW

File: unit_converter/__init__.py [hunk 1]
```diff
@@ -24,19 +24,18 @@
...
 
-import albert
 import inflect
 import pint
+from albert import *
 
-
-md_iid = '2.0'
+md_iid = "2.1"
 md_version = "1.4"
```
Comment (ManuelSchneid3r) [inline unit_converter/__init__.py:32]: please update the version if you change the extension.

Comment (Pete-Hamlin) [reply unit_converter/__init__.py]: Good catch, fixed in 1d34d579a1ae8f56f954e201d61f5c021131de55

**Comment by Pete-Hamlin** [comment]
Been having some weird performance issues since having this plugin activated.

Going to do a little more R&D before I post up for review, so going to close this for now.

**Comment by Pete-Hamlin** [comment]
OK, have been testing this one for the past month and can confirm that the performance issues were unrelated to this one.

Confident that this is now ready to merge, however may now conflict with https://github.com/albertlauncher/python/pull/195

**Comment by Pete-Hamlin** [comment]
@ManuelSchneid3r after discussing with @ismasou I think we're good to go ahead and merge this branch and close https://github.com/albertlauncher/python/pull/195

**Comment by fgibaux** [comment]
do I understand correctly that the unit converter is not coming back as both #195 and this have been closed without being merge?

**Comment by ManuelSchneid3r** [comment]
Happened accidentally. the change of the default branch automatically closed it.

**Comment by ManuelSchneid3r** [comment]
This issue was not intentionally but automatically closed because I renamed the master branch to main. If you are still interested in getting it merged please reopen against `main`.

**Comment by hfr-hfr** [comment]
@ManuelSchneid3r , is there anything we can do to help this getting merged? Is there any issue with it?

**Pull request closed without merging**